src/Control/Lens/Unsound.hs

```diff
@@ -0,0 +1,92 @@
+{-# LANGUAGE CPP #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE UndecidableInstances #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE FunctionalDependencies #-}
+{-# OPTIONS_GHC -fno-warn-warnings-deprecations #-}
+
+#ifndef MIN_VERSION_mtl
+#define MIN_VERSION_mtl(x,y,z) 1
+#endif
+
+#if __GLASGOW_HASKELL__ < 708
+{-# LANGUAGE Trustworthy #-}
+#endif
+
+{-# LANGUAGE RankNTypes #-}
+
+-------------------------------------------------------------------------------
+-- |
+-- Module      :  Control.Lens.Unsafe
```

Comment:
I'm like 995 sure this should be Control.Lens.Unsound

Reply:
fixed